src/Client.js

```diff
@@ -64,6 +66,16 @@ class Client {
 
     this.amqp.startConsumer(exchange, routingKeys, queue, onMessage);
   }
+
+  async createUser(email, password) {
+    const body = { email, password };
+    return await this.http.post('users', body);
+  }
+
+  async createToken(email, password) {
+    const body = { email, password };
+    return await this.http.post('tokens', body);
+  }
 }
```

Comment:
We must now pass `authToken` as a parameter to these operations :point_down: 

```suggestion
  async createUser(email, password, authToken) {
    const body = { email, password };
    return await this.http.post('users', body, authToken);
  }

  async createToken(email, password, authToken) {
    const body = { email, password };
    return await this.http.post('tokens', body, authToken);
  }
}
```

Reply:
Done!